src/Cache/CacheKeyAndTags.php

```diff
@@ -11,8 +11,9 @@ interface CacheKeyAndTags
      *
      * @param  int|string|null $id
      * @param  array<string, mixed> $args
+     * @param  array<int, string> $path
      */
-    public function key(?Authenticatable $user, bool $isPrivate, string $parentName, $id, string $fieldName, array $args): string;
+    public function key(?Authenticatable $user, bool $isPrivate, string $parentName, $id, string $fieldName, array $args, array $path): string;
```

Comment:
```suggestion
    public function key(?Authenticatable $user, bool $isPrivate, array $args, ResolveInfo $resolveInfo): string;
```

That way we can avoid future breakage and simply have all relevant context available.